dev/test-wheel.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
+
+echo "=== test-wheel.sh ==="
+
+# Build the wheel first
+echo "Building the wheel: start"
+./dev/build.sh
+echo "Building the wheel: done"
```

Comment:
@adam-narozniak this should not be part of the test script (otherwise re re-build the wheel every time we run the wheel tests)

Reply:
@danieljanes I thought it should be done like it. 
But I can change it, and require the build by a user. I can check if the directory is empty, yet it won't be resilient to testing an old wheel accidentally.